src/dnddiceroller/DiceTower.java

```diff
@@ -14,15 +14,54 @@
  * they reach the tray at the bottom
  * @author Paul Scarrone
  */
+
 public class DiceTower {
-  final int PANEL_COUNT = 3;
-  List<Die> dice;
+  final int PANEL_COUNT = 3;    // number of panels in dice tower
+  List<Die> dice;               // arraylist of die objects
+  private int dieNum;           // number rolled by a die
+  private int trayValue;        // value of all dice rolled
 
+  /** 
+   * constructor
+   * no params
+   */
   public DiceTower() {
-	this.dice = new ArrayList();
+    this.dice = new ArrayList();
+    dieNum = 0;
+    trayValue = 0;
   }
   
+  /** 
+   * constructor
+   * @param dice  arraylist of die objects
+   */
   public DiceTower(List dice) {
-	this.dice = dice;
+    this.dice = dice;
+    dieNum = 0;
+    trayValue = 0;
+  }
+  
+  /** 
+   * dropDice   drop die objects in arraylist through dice tower to roll
+   * no params  uses class fields
+   * returns nothing
+   */
+  public void dropDice(){
+      // roll the dice as many times as there are panels in dice tower
+      for (int cubes = 0; cubes < dice.size(); cubes++){
+        for (int rolls = 0; rolls < PANEL_COUNT; rolls++) {
+            this.dice.get(cubes).roll();
+        }
+        trayValue += this.dice.get(cubes).getValue();
+      }
+  }
+  
+  /** 
+   * trayValue  adds together the roll results of die objects in arraylist
+   * @return    sum of roll results
+   */
+  public int trayValue(){
+    // return sum of values of dice
+      return trayValue;
   }
 }
```

Comment:
You do such a nice job commenting.  It really makes it so easy to follow your code.